src/main/java/eu/xtrf/html2pdf/server/converter/service/Html2PdfConverterServiceImpl.java

```diff
@@ -61,25 +81,60 @@ private static String htmlToXhtml(String inputHTML) {
         return replaceHtmlEntitiesNamesWithNumbers(document.html());
     }
 
-    private static ITextRenderer prepareRenderer(String html, String resourcesPath) throws IOException {
-        ITextRenderer renderer = new ITextRenderer();
+    private ITextRenderer prepareRenderer() throws IOException {
```

Comment:
co myslisz zeby metody do ladowania czcionek itp. wyniesc do innej klasy, i ten renderer podawac jako parametr do kosntruktora?
Poprawiloby to testowalnosc itp.
Pewnie to ladowanie czcionek jest jednorazowe , to fajniebyloby to miec poza serwisem, bo ten sie pewnei bedzie mocniej rozrastal, a kod ladowania czcionek pewnie nie bedzie sie tak bardzo zmeinial, i nie trzeba bedzie sie w niego wczytywac jak bedzie w osobnej klasie.